mobile/src/common/types/user/user-group-creation.dto.type.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactElement } from 'react';
+
+type UsersGroupCreationDto = {
```

Comment:
same as about groups
```suggestion
type UsersTableRow = {
```